sources/packages/web/src/composables/useApplication.ts

```diff
@@ -22,5 +27,23 @@ export function useApplication() {
     }
   };
 
-  return { mapApplicationChipStatus };
+  const mapApplicationDetailHeader = (
+    application: Required<ApplicationDetailHeader>,
+  ): Record<string, OfferingIntensity | string> => {
+    const { dateOnlyLongString } = useFormatters();
+
+    return {
+      "Application #": application.applicationNumber,
+      Institution: application.applicationInstitutionName,
+      "Study dates":
+        application.applicationStartDate && application.applicationEndDate
+          ? `${dateOnlyLongString(
+              application.applicationStartDate,
+            )} - ${dateOnlyLongString(application.applicationEndDate)}`
+          : "-",
```

Comment:
Is that a requirement that when there is no data for the header component we must show - or instead not populate the keyValue itself?

Reply:
` instead not populate the keyValue itself` i didn't understand. what do you mean?

Reply:
checked with @hellolynn-tbtb, she asked to keep "-" for now